packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.story.tsx

```diff
@@ -1,70 +1,65 @@
 import { action } from '@storybook/addon-actions';
+import { useArgs } from '@storybook/client-api';
+import { Story } from '@storybook/react';
 import React from 'react';
 
 import { RefreshPicker } from '@grafana/ui';
 
-import { DashboardStoryCanvas } from '../../utils/storybook/DashboardStoryCanvas';
-import { StoryExample } from '../../utils/storybook/StoryExample';
-import { UseState } from '../../utils/storybook/UseState';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
-import { HorizontalGroup } from '../Layout/Layout';
+
+import { Props } from './RefreshPicker';
+import mdx from './RefreshPicker.mdx';
 
 export default {
   title: 'Pickers and Editors/RefreshPicker',
   component: RefreshPicker,
   decorators: [withCenteredStory],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  args: {
+    isLoading: false,
+    isLive: false,
+    width: 'auto',
+    text: 'Run query',
+    tooltip: 'Run query',
+    value: '1h',
+  },
+  argTypes: {
+    intervals: {
+      control: {
+        type: 'select',
+      },
+    },
+  },
 };
 
-export const Examples = () => {
+export const Examples: Story<Props> = (args) => {
+  const [, updateArgs] = useArgs();
   const intervals = ['5s', '10s', '30s', '1m', '5m', '15m', '30m', '1h', '2h', '1d'];
```

Comment:
let's move this into the `args` and use `args.intervals` to pass it to the component